### PR TITLE
docs: add cloud metrics/exporters guide (fixes #358)

### LINKED ISSUE #358: Write guide for adding credentials and configuring/deploying exporters
Write a guide covering:
- (brief, ideally external link) Getting credentials from AWS/GCP
- Adding those credentials into opstrace via curl
- Adding stackdriver/cloudwatch exporters into opstrace via curl, using a basic config example
- Observing the metrics appearing in opstrace

### PR DESCRIPTION
**Pull request by nickbp**
First draft, likely needs restructuring to be a proper guide. Anything that sticks out as missing?

Signed-off-by: Nick Parker <nick@opstrace.com>

# Have you...

* [ ] Discussed your change with a project contributor in an issue yet?
* [ ] Added an explanation of what your changes do?
* [ ] Written new tests for your changes?
* [ ] Thought about which docs need updating?

### CODE REVIEW

File: docs/guides/user/collecting-cloud-metrics.md [hunk 1]
```diff
@@ -0,0 +1,311 @@
+# Cloud Metrics
+
+Prometheus supports a wide range of exporters for getting metrics out of other systems and into Prometheus.
+We have recently added support for adding exporters for AWS CloudWatch and GCP Stackdriver directly into Opstrace.
```
Comment (jgehrcke) [inline docs/guides/user/collecting-cloud-metrics.md:4]: "We have recently added" is probably not relevant here, Towards shortening things, things like that can go : )

**Review by triclambert — APPROVED** [review]
I think this fine to ship and iterate on from there.

File: docs/guides/user/collecting-cloud-metrics.md [hunk 2]
```diff
@@ -0,0 +1,311 @@
...
+Credentials may be referenced by one or more exporters under the same tenant.
+Updating a credential will automatically restart exporters using that credential.
+Each credential within a tenant has a unique name.
+
+#### List all credentials
+
+Returns credential information for all credentials configured for the tenant.
+Each entry includes credential metadata, while the credential content is not included.
+
+##### Example
```
[the remark this answers is not in the document]
Comment (nickbp) [reply docs/guides/user/collecting-cloud-metrics.md]: Removed all of the `Example` subheaders, they are indeed redundant

File: docs/guides/user/collecting-cloud-metrics.md (hunk 1 above)
Comment (nickbp) [reply docs/guides/user/collecting-cloud-metrics.md]: Oh whoops, might have been left over from the blog post. Fixed

File: docs/guides/user/collecting-cloud-metrics.md [hunk 3]
```diff
@@ -0,0 +1,311 @@
...
+config:
+  monitoring.metrics-type-prefixes:
+  - compute.googleapis.com/instance/cpu
+' | curl -v -H "Authorization: Bearer $(cat tenant-api-token-dev)" --data-binary @- http://me.opstrace.io/api/v1/exporters/
+```
+
+#### Delete exporter
+
+Deletes the exporter, tearing it down and ending the export of the configured metrics.
+Any referenced credential is not deleted.
```
[the remark this answers is not in the document]
Comment (nickbp) [reply docs/guides/user/collecting-cloud-metrics.md]: Good point, added mention of this as well

**Comment by triclambert** [comment]
Feel free to merge ASAP once review feedback has been addressed.

**Pull request merged**